src/components/modals/tickets/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { BoxContainer } from './styled';
+import { ITicketState } from '../../../types/tickets';
+import { Modal } from '@mui/material';
+import TicketsForm from '../../form/tickets';
+import { memo } from 'react';
+interface IProps {
+  toggleModal: () => void;
+  initialValues: ITicketState | null;
+  isEdit?: boolean;
+  apiUrl: string;
+}
+
+const TicketsModal = ({ toggleModal, initialValues, isEdit, apiUrl }: IProps) => {
+  return (
+    <Modal open={true} onClose={toggleModal}>
```

Comment:
Would it be better to pass `isOpen` props to trigger `<Modal open={isOpen}`?

Reply:
fix: passed isModalOpen